.env.common

```
@@ -15,7 +15,7 @@ SMTP_AUTH=true
 SMTP_USER=prusek@utia.cas.cz
 SMTP_PASS=M1i2c3h4a5l6
 FROM_EMAIL=prusek@utia.cas.cz
-FROM_NAME="Cell Segmentation Platform"
+FROM_NAME="SpheroSeg"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)